holoviews/operation/element.py

```diff
@@ -571,6 +568,15 @@ def _process(self, element, key=None):
         # if any data is a datetime, transform to matplotlib's numerical format
         data_is_datetime = tuple(isdatetime(arr) for k, arr in enumerate(data))
         if any(data_is_datetime):
+            if any(data_is_datetime[:2]) and self.p.filled:
+                raise RuntimeError("Datetime spatial coordinates are not supported "
+                                   "for filled contour calculations.")
+
+            try:
+                from matplotlib.dates import num2date, date2num
+            except ImportError:
+                raise ImportError("contours operation using datetimes requires matplotlib.")
```

Comment:
```suggestion
                raise ImportError("contours operation using datetimes requires matplotlib.") from None
```

No need to see two import errors.

Reply:
Nice. I didn't know this syntax.